tests/suites/test_suite_ssl.function

```diff
@@ -6,6 +6,39 @@
 #include <mbedtls/certs.h>
 #include <mbedtls/timing.h>
 
+typedef struct handshake_test_options
+{
+    const char *cipher;
+    int version;
+    int pk_alg;
+    data_t *psk_str;
+    int dtls;
+    int serialize;
+    int mfl;
+    int cli_msg_len;
+    int srv_msg_len;
+    int expected_cli_fragments;
+    int expected_srv_fragments;
+    int renegotiate;
+    int legacy_renegotiation;
+} handshake_test_options;
+
+void init_handshake_options( handshake_test_options *opts )
+{
+  opts->cipher = "";
+  opts->version = MBEDTLS_SSL_MINOR_VERSION_3;
```

Comment:
Is this really a good default value for version?

Reply:
This is a version used by default in ssl_client2 and ssl_server2, so I decided to reuse it here.

Reply:
OK. I still question it as a default for us to use in testing, both here and in ssl_client2 and ssl_server2, since a majority of our users probably don't use that version.